backend/apid/routers/authentication.go

```diff
@@ -194,6 +197,9 @@ func (a *AuthenticationRouter) token(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	// Make sure to the 'system:users' group is present
+	user.Groups = append(user.Groups, "system:users")
+
```

Comment:
Wouldn't this ultimately create an ever growing number of `system:users` entries in the user's group list?
Same thing in the previous hunk in `login()`.

Reply:
No because we never update the actual user in the store with this group; it's only injected in the user claims; it will never show up in the `sensuctl` commands and if you update a user group, it will be re-injected here once we renew the token.